week01/136. Single Number.ts

```diff
@@ -0,0 +1,25 @@
+function singleNumber(nums: number[]): any {
+  let hashTable: any = {};
+  nums.forEach((item) => {
+    if (item in hashTable) hashTable[item] += 1;
+    else hashTable[item] = 1;
+  });
+  for (const item in hashTable) {
+    if (hashTable[item] === 1) return parseInt(item);
+  }
+  //   hashTable;
+  //   return nums.reduce((prev, curr) => prev ^ curr);
+  //   return 0;
+}
```

Comment:
Not acceptable bhaia. :) This is pure brute force. Interviewer will not allow this solution. XOR is the preferable way here.

Reply:
ok, bhai.